src/lib/ngsi9/DiscoverContextAvailabilityRequest.cpp

```diff
@@ -76,7 +76,7 @@ std::string DiscoverContextAvailabilityRequest::check(ConnectionInfo* ciP, Reque
   {
     response.errorCode.fill(SccContextElementNotFound);
   }
-  else if (((res = entityIdVector.check(ciP, DiscoverContextAvailability, indent, predetectedError, restrictions))                      != "OK") ||
+  else if (((res = entityIdVector.check(DiscoverContextAvailability, indent))                                                      != "OK") ||
```

Comment:
As discussed at skype, let's consider style-check aspect in the last PR in the "refactor cycle". NTC